bin/test_e2e

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -e
+
+#
+# Remote Integration Test Runner
+#
+# This runs only e2e features, which are those dependent on
+# access to a remote PCF instance. Run this script from the `bin`
+# directory with `summon ./test_e2e`, with
+# proper summon credentials.
+#
+
+# First, check for summon variables
+: ${CF_API_ENDPOINT?"Error: Need to set CF_API_ENDPOINT"}
+
+cd "$(dirname $0)"
+. ./utils
```

Comment:
CodeClimate isn't smart here, and complains about `Not following` (can't find `./utils`).
You might need to add an exception comment above this line:
```
# shellcheck disable=SC1091
```